packages/bpk-component-calendar/src/BpkCalendarGridTransition.js

```diff
@@ -157,7 +161,7 @@ class BpkCalendarGridTransition extends Component {
           onTransitionEnd={this.onMonthTransitionEnd}
         >
           {this.state.months.map((m, index) =>
-            isWithinRange(m, min, max) ? (
+            min && max && isWithinRange(m, { start: min, end: max }) ? (
```

Comment:
`isWithinRange` now throws an exception if you don't pass in valid dates, so we use this check to make sure we don't do that.